packages/design-system/src/components/web-components/ds-modal-dialog/ds-modal-dialog.tsx

```diff
@@ -53,14 +54,14 @@ const Wrapper = ({
     isOpen={parseBooleanAttr(isOpen)}
     size={isPossibleValue(size, availableSizes) ? size : undefined}
     {...otherProps}
-    analytics={parseBooleanAttr(analytics)}
+    analytics={analytics === undefined ? undefined : analytics !== 'false'}
```

Comment:
Seems like we should consider whether we want analytics to be opt-in, or on by default. Seems like we've been doing on by default, which is why this fails, right?

Reply:
Opt in meaning that users have to pass true to turn on analytics, just to be explicit. If opt in were the case, would they need to supply additional configuration info, or could they use the defaults provided by the config function?

Reply:
This line change makes it behave like our React components behave, where providing a value for `analytics` (whether `true` or `false`) overrides the application-level default. We do actually require teams to opt in, but they can either opt in at the application level by calling `config()` and changing the per-component settings or by passing `analytics={true}` to a particular instance of a component.

Reply:
That is, this should now accurately reflect the [analytics setting behavior we describe here](https://design.cms.gov/components/analytics/?theme=core#turning-it-on-and-off)

Reply:
I just noticed that the code looks different for [ds-alert and ds-button](https://github.com/CMSgov/design-system/blob/pwolfert/fix-ds-modal-dialog-analytics/packages/design-system/src/components/web-components/ds-alert/ds-alert.tsx#L28) but it should have roughly the same result. Maybe I should change it here to be consistent, though.